.github/workflows/test.yml

```diff
@@ -8,7 +8,18 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  llvm-toolchain:
+    runs-on: ubuntu-latest
+    outputs:
+      llvm: ${{ steps.llvm-toolchain-impl.outputs.version }}
+    steps:
+      - id: llvm-version
+        uses: libbpf/ci/get-llvm-version@master
+      - id: llvm-toolchain-impl
+        shell: bash
+        run: echo "::set-output name=version::llvm-${{ steps.llvm-version.outputs.version }}"
```

Comment:
It's a bit cumbersome, but basically `llvm-toolchain` is the name of the job. It has one output, `llvm`. That basically just "forwards" to `llvm-toolchain-impl.version` (because it can't just be evaluated inline there). `llvm-toolchain-impl` is just a `bash` script that echos `echo "::set-output name=version::llvm-${{ steps.llvm-version.outputs.version }}"` to stdout. That's some kind of structured data, as that's how GitHub Actions communicate output. `::set-output name=` is the "function" that tells the job to set the output. What follows is a `::` separated tuple of the name of the output, `version`, and its content. So in our case the content is just an interpolated string, `llvm-${{ steps.llvm-version.outputs.version }}`. Here, `steps.llvm-version.outputs.version` is the output of the `get-llvm-version` action introduced [here](https://github.com/libbpf/ci/pull/27) and run as the step just before this one. So basically, we just prefix that with `llvm-`. Does that explanation cover what you asked for?